packages/ipfs/test/http-api/inject/block.js

```diff
@@ -60,7 +60,9 @@ describe('/block', () => {
       return testHttpMethod('/api/v0/block/put')
     })
 
-    it('returns 400 if no node is provided', async () => {
+    // Intermittent timeouts
+    // https://github.com/ipfs/js-ipfs/issues/3105
+    it.skip('returns 400 if no node is provided', async () => {
```

Comment:
This was failing because hapi doesn't support node 14 yet. Ugh. I've disabled node 14 in master, it should be safe to unskip now.

Reply:
@achingbrain Here is the result of previous commit, which was merge with master
https://github.com/ipfs/js-ipfs/runs/788842730

Reply:
Never mind I had to refresh browser